README.rst

```diff
@@ -104,6 +104,14 @@ contributions!
 See the `developer docs`_ and the `contribution docs`_ for more information.
 
 
+Security
+========
+
+If you think you discovered a security flaw, please report it to security@crate.io.
```

Comment:
```suggestion
If you think you discovered a security flaw, please report it to security@cratedb.com.
```

Reply:
No sure if we are using cratedb.com for email already. I don't think so, but I will ask.

Reply:
@wmKvXo confirmed it. `crate.io` it still is.